### ISSUE TITLE
Ketting server jar doesnt download

### ISSUE THREAD

**Issue by Willplayz112** [itzg/docker-minecraft-server#2609]
### Describe the problem

When starting a ketting server jar the URL does not work because it returns a 404 error page

### Container definition

```
i ran it in portainer

```

### Container logs

```

[mc-image-helper] 16:22:59.691 ERROR : 'get' command failed. Version is 1.37.8
me.itzg.helpers.get.RequestFailedException: Failed to download https://github.com/kettingpowered/Ketting-1-20-x/releases/download/e4381fa/ketting-1.20.1-1.20.1-forge-47.2.16-e4381fa-server.jar: status code: 404, reason phrase: Not Found
	at me.itzg.helpers.get.GetCommand.resolveExpectedFilename(GetCommand.java:364)
	at me.itzg.helpers.get.GetCommand.needsDownload(GetCommand.java:329)
	at me.itzg.helpers.get.GetCommand.processUrisForDirectory(GetCommand.java:303)
	at me.itzg.helpers.get.GetCommand.call(GetCommand.java:198)
	at me.itzg.helpers.get.GetCommand.call(GetCommand.java:55)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at me.itzg.helpers.McImageHelper.main(McImageHelper.java:157)
Caused by: org.apache.hc.client5.http.HttpResponseException: status code: 404, reason phrase: Not Found
	at me.itzg.helpers.http.FilenameExtractor.extract(FilenameExtractor.java:73)
	at me.itzg.helpers.http.DeriveFilenameHandler.handleResponse(DeriveFilenameHandler.java:21)
	at me.itzg.helpers.http.DeriveFilenameHandler.handleResponse(DeriveFilenameHandler.java:10)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:247)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:188)
	at org.apache.hc.client5.http.impl.classic.CloseableHttpClient.execute(CloseableHttpClient.java:162)
	at me.itzg.helpers.get.GetCommand.resolveExpectedFilename(GetCommand.java:356)
	... 13 common frames omitted
[init] ERROR: failed to download Ketting server jar from https://github.com/kettingpowered/Ketting-1-20-x/releases/download/e4381fa/ketting-1.20.1-1.20.1-forge-47.2.16-e4381fa-server.jar


```

**Comment by Willplayz112** [comment]
all you really need to do it change the URL

**Comment by itzg** [comment]
> all you really need to do it change the URL
> 
> 

PRs are welcome

**Comment by itzg** [comment]
@aruneko also perhaps you can look into this since it apparently already changed since #2535

**Comment by aruneko** [comment]
It's not a bug. You give wrong `FORGE_VERSION` and `KETTING_VERSION` environment variable. This causes the 404 error.

The available value of the `FORGE_VERSION` is `48.1.0` against to the Ketting release `e4381faa`. And the length of `KETTING_VERSION` is 8, not 7 since the value comes from a abbreviated git commit hash.

**Comment by itzg** [comment]
Will be superseded by #2650
